core/src/main/scala/scalaz/Cokleisli.scala

```diff
@@ -45,6 +45,9 @@ sealed abstract class CokleisliInstances0 {
     new CokleisliProfunctor[F] {
       def F = implicitly
     }
+
+  implicit def cokleisliBindRec[F[_], R]: BindRec[Cokleisli[F, R, ?]] =
+    new CokleisliBindRec[F, R] {}
 }
```

Comment:
`cokleisliBindRec[F[_], R]: BindRec[Cokleisli[F, R, ?]]` and `cokleisliMonad[F[_], R]: Monad[Cokleisli[F, R, ?]]` are same constraints.

should we change `cokleisliInstance[F[_], R]: BindRec[Cokleisli[F, R, ?]] with Monad[Cokleisli[F, R, ?]] =` ?